Kudu.Core/Kube/SyncTriggerAuthenticator.cs

```diff
@@ -21,27 +21,27 @@ public async static Task<bool> AuthenticateCaller(Dictionary<string, IEnumerable
             {
                 return false;
             }
-
+            Console.WriteLine("**** SyncTriggerAuthenticator Started ***");
             //If there's no encryption key in the header return true
             if (!headers.TryGetValue(SiteTokenHeaderName, out IEnumerable<string> siteTokenHeaderValue))
             {
                 return false;
             }
-
+            Console.WriteLine("**** SyncTriggerAuthenticator finish TryGetValue ***");
             //Auth header value is null or empty return false
             var funcAppAuthToken = siteTokenHeaderValue.FirstOrDefault();
             if (string.IsNullOrEmpty(funcAppAuthToken))
             {
                 return false;
             }
-
+            Console.WriteLine("**** SyncTriggerAuthenticator funcAppAuthToken ***");
```

Comment:
are these Console.WriteLine needed? Can they use a logger of some sort?

Reply:
Thank you @ahmelsayed This line is just for temporary debugging, However, I'd like to use Logger. 
I found `ITracer` is injected, so that I tried to use it, However, it doesn't show anything on the console log. Do you know why it doesn't show the console log? Do we need to configure for it?

Reply:
Are you referring to `ILogger` ? If yes, you need to explicitly add console logging provider - https://docs.microsoft.com/en-us/aspnet/core/fundamentals/logging/?view=aspnetcore-5.0

Reply:
As we discussed on the standup, I remove the Console logs. 
I'll update the logs for production on this issue. https://github.com/Azure-App-Service/KuduLite/issues/202